apps/nextjs-website/src/app/auth/change-password/page.tsx

```diff
@@ -27,33 +27,36 @@ const ChangePassword = () => {
   const username = searchParams.get('username') || '';
   const code = searchParams.get('code') || '';
 
+  const [state, setState] = useState<State>(State.loading);
+  const [submitting, setSubmitting] = useState(false);
   const [resetPasswordSteps, setResetPasswordSteps] = useState(
     ResetPasswordSteps.CHANGE_PASSWORD
   );
-  const [password, setPassword] = useState('');
-  const [state, setState] = useState<State>(State.loading);
-
-  const onChangePassword = useCallback(async () => {
-    const success = await Auth.forgotPasswordSubmit(
-      username,
-      code,
-      password
-    ).catch(() => {
-      setState(State.errorLink);
-      return false;
-    });
-
-    if (!success) return;
 
-    setResetPasswordSteps(ResetPasswordSteps.CHANGE_PASSWORD_SUCCESS);
-  }, [code, password, username]);
+  const onChangePassword = useCallback(
```

Comment:
Maybe you should rename this const to `onPasswordChanged` (I think it's similar to this [comment](https://github.com/pagopa/developer-portal/pull/694/files/368ed67f1feaeef2c613a347a1a95fd86ad0752a#r1530656419))